integration-tests/rest-tests-jaxrs/src/test/groovy/org/hawkular/metrics/rest/ExpressionTagQueryITest.groovy

```diff
@@ -170,6 +170,30 @@ class ExpressionTagQueryITest extends RESTTest {
         headers: [(tenantHeaderName): tenantId])
       assertEquals(200, response.status)
       assertMetricListById(response.data, 'm3', 'm4', 'm5', 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b"],
+        headers: [(tenantHeaderName): tenantId])
+      assertEquals(200, response.status)
+      assertMetricListById(response.data, 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b = c.d"],
+        headers: [(tenantHeaderName): tenantId])
+      assertEquals(200, response.status)
+      assertMetricListById(response.data, 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b = 'c.d'"],
+        headers: [(tenantHeaderName): tenantId])
+      assertEquals(200, response.status)
+      assertMetricListById(response.data, 'm6')
+
+      response = hawkularMetrics.get(path: "metrics",
+        query: [tags: "a.b = 'c.*'"],
```

Comment:
Same here, if the intention was to test ``startsWith(c.)`` then this doesn't do that, but instead of ``startsWith(c)``